openassessment/xblock/static/js/lib/table_builder.js

```diff
@@ -0,0 +1,356 @@
+"use strict"
+
+$.fn.oraTableBuilder = function ($config) {
+    const DIV = 'div', CHECKBOX = 'checkbox', ROW = 'row', COLUMN = 'column';
+    const CHECKBOX_SELECTOR = 'input[type="checkbox"]', CONTENTENDITABLE_SELECTOR = '[contenteditable]';
+    const _this = this
+    var change_cb = function (table) {
+        console.info("Please overrider change in configuration object");
+    }
+    if ($config && $config.change) {
+        change_cb = $config.change;
+    } else {
+        $config.change = change_cb;
+    }
+
+    function plugin() {
+        this._this = _this;
+        const __this = this;
```

Comment:
I don't quite understand this.
If you wanna make a new object then please use `Object.assign({}, _this)`.

Reply:
@RanaWaqar7550 I want to reflect changes to the context of parent closure here that's why I cannot make the new object.